src/components/details.js

```diff
@@ -79,16 +80,18 @@ class HeadingWithDetail extends Component {
               <Message>{ details }</Message>
             </Transition>
           }>
+
           <div className={ `has-details` }>{ top }</div>
+
         </Surrounder>
       </div>
     );
   };
-};  // End <HeadingWithDetail>
+};  // Ends <HeadingWithDetail>
 
 
-/** @todo Have rows that can have details. Maybe other things too. */
-class RowWithDetail extends Component {};  // <End RowWithDetail>
+/* @todo Have rows that can have details. Maybe other things too. */
+class RowWithDetail extends Component {};
```

Comment:
I know this is existing code and not a part of this PR, but I am confused why we have an empty class at all. Is this event used anywhere? It doesn't look like it is. We can open a separate issue for this if necessary.

Reply:
Yeah, this can be a separate issue. I've been debating about that. I think the first one may be generic enough to be used with a row item as well, so we may just need to change the name.

@todo 

It'd be great to have an issue about this.